src/app/shell/personal-cabinet/shared-cabinet/messages/chat/chat.component.html

```diff
@@ -2,46 +2,36 @@
   <div class="chat-card__header header">
     <mat-icon class="header__icon" (click)="onBack()">arrow_back</mat-icon>
     <div class="header__companion-info">
-      <h4 class="header__title">
-        {{
-          userRole === Role.parent
-            ? chatRoom?.workshop.title
-            : (chatRoom?.parent | getFullName)
-        }}
+      <h4 class="header__title" *ngIf="!!chatRoom">
+        {{ userRole === Role.parent ? chatRoom?.workshop?.title : (chatRoom?.parent | getFullName) }}
```

Comment:
i know this is not your changes, but mb we should remove logic from the template

Reply:
These are my changes from another branch)
Okey, I'll see how best to do it